tests/ui/macros/macro-rules-attr-error.stderr

```diff
@@ -0,0 +1,22 @@
+error: local_attr: struct S;
+  --> $DIR/macro-rules-attr-error.rs:5:9
+   |
+LL |         compile_error!(concat!("local_attr: ", stringify!($($body)*)));
+   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+...
+LL |     #[local_attr]
+   |     ------------- in this attribute macro expansion
+   |
+   = note: this error originates in the attribute macro `local_attr` (in Nightly builds, run with -Z macro-backtrace for more info)
+
+error: invoked macro has no invocation rules
```

Comment:
Strange wording, the macro has rules, just not fn-like rules.

A bit of terminology (from the compiler, didn't check the reference):
- Macro invocation - invocation of any macro - fn-like, or attr, or derive
- Macro call - invocation of a fn-like macro

Reply:
The reference seemed to use the term "invoke" specifically for bang invocation, not attributes or derives. I can come up with another way to describe this, though.